util/chunk/codec.go

```diff
@@ -105,6 +105,7 @@ func (c *Codec) DecodeToChunk(buffer []byte, chk *Chunk) (remained []byte) {
 	return buffer
 }
 
+// Todo: Optimize all data is null.
```

Comment:
Comments should be started with the function name in golang.

Reply:
Update.